### ISSUE TITLE
Support requiring fingerprint to view codes

### ISSUE THREAD

**Issue by trs80** [freeotp/freeotp-android#89]
Since Android 6.0, you can require a fingerprint authentication, it'd be nice to use this when opening or switching to the app. https://developer.android.com/about/versions/marshmallow/android-6.0.html#fingerprint-authentication

**Comment by grmpyninja** [comment]
It's for sure an awesome idea (as a fallback PIN would be enough) to simply protect against an attack when you left your phone unlocked and somebody wants to use your OTP codes to get access to your accounts.

+1 for fingerprint and PIN as a fallback

**Comment by Niek** [comment]
+1
This app could really be awesome if fingerprint/PIN support was added (as well as external backup).

**Comment by synthead** [comment]
This would be an awesome feature!

**Comment by mxdlx** [comment]
Used to have this on iOS, so it'd be nice to have this on Android.

**Comment by michag86** [comment]
Is this really fixed?
I cannot find any option to enable fingerprint or pin.

I'm using FreeOTP version 2.0.4

**Comment by justin-stephenson** [comment]
@michag86 No, the logic was partially implemented but the full functionality still needs to be reviewed and tested with UI changes for Biometrics. Apologies for the confusion.

**Comment by michag86** [comment]
@justin-stephenson can you please reopen this issue?

**Comment by justin-stephenson** [comment]
> @justin-stephenson can you please reopen this issue?

Done
